packages/design-system/src/styles/settings/variables/_core-scss-to-css.map.scss

```diff
@@ -0,0 +1,15 @@
+$grid-columns: 12 !default;
```

Comment:
maybe these filenames could be something more like `layout-tokens.scss` ? The map isn't really relevant anymore with these.